src/app/my-account/my-account.component.ts

```diff
@@ -67,7 +67,7 @@ export class MyAccountComponent implements OnInit {
             color: 'brand-afa-pink',
             image: new URL(environment.donateUriPrefix + '/assets/images/peach-texture.jpg')
           },
-          headerText: "Your Regular\u00A0Giving", // u00A0 = non-breaking space. We prefer the break after "Your".
```

Comment:
Think this looks good. We'd like to encode that we prefer the break after Your but a break after Regular is also allowed, but seems there's no way to do that https://stackoverflow.com/a/18240699/2526181